### ISSUE TITLE
Honkai Star Rail: Caches update scan results in error

### ISSUE THREAD

**Issue by seriaati** [neon-nyan/Collapse#163]
# Launcher Version
1.70.8 Preview
# Reproduce
1. Select game "Honkai: Star Rail" and region "Global"
2. Click on the menu button in the top left
3. Select "Caches Update"
4. Click "Quick Scan"
# Error Log
```
System.UriFormatException: Invalid URI: The format of the URI could not be determined.
   at System.Uri.CreateThis(String, Boolean, UriKind, UriCreationOptions& )
   at System.Uri..ctor(String uriString)
   at Hi3Helper.Http.Http.InitializeSingleSession(Nullable`1 OffsetStart, Nullable`1 OffsetEnd, Boolean IsFileMode, Stream _Stream, Boolean IgnoreOutStreamLength)
   at Hi3Helper.Http.Http.<>c__DisplayClass66_0.<Download>b__0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread, ExecutionContext, ContextCallback, Object)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread, ExecutionContext, ContextCallback, Object)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task&, Thread )
--- End of stack trace from previous location ---
   at Hi3Helper.Http.Http.Download(String URL, Stream Outstream, Nullable`1 OffsetStart, Nullable`1 OffsetEnd, CancellationToken ThreadToken, Boolean IgnoreOutStreamLength)
   at Hi3Helper.EncTool.Parser.AssetMetadata.SRDispatcherInfo.ParseArchive()
   at Hi3Helper.EncTool.Parser.AssetMetadata.SRDispatcherInfo.Initialize(CancellationToken threadToken, String persistentDirectory, String regionName)
   at Hi3Helper.EncTool.Parser.AssetMetadata.SRMetadata.Initialize(CancellationToken threadToken, String regionName, String persistentDirectory, Boolean forceInitialize)
   at CollapseLauncher.StarRailCache.Fetch(CancellationToken token)
   at CollapseLauncher.StarRailCache.CheckRoutine()
   at CollapseLauncher.Interfaces.ProgressBase`2.TryRunExamineThrow(Task`1 action)
   at CollapseLauncher.StarRailCache.StartCheckRoutine(Boolean useFastCheck)
   at CollapseLauncher.Pages.CachesPage.RunCheckRoutine(Object sender, Boolean isFast, Boolean isMainButton)
   ```

**Comment by bagusnl** [comment]
Related to #162 
Work is in progress by neon

**Comment by Cryotechnic** [comment]
Fixed in [bf8153f](https://github.com/neon-nyan/Hi3Helper.EncTool/commit/bf8153f99491feabdcf3c71861adf887e45a3d3d). Will ship in v1.70.X if it has not already been shipped. Going to close this now as it is a duplicate issue. Please refer to #162 for more information & tracking.
